packages/shared/components/popups/Transaction.svelte

```diff
@@ -41,14 +36,30 @@
         return isFiat ? `${fiatAmount} (${iotaAmount})` : `${iotaAmount} (${fiatAmount})`
     }
 
+    $: accountOverview = $participationOverview?.find((apo) => apo?.accountIndex === $selectedAccount.index)
+    $: isAccountVoting = Object.values(accountOverview?.trackedParticipations)?.find((tp) =>
+        tp?.find((p) => p?.endMilestoneIndex === 0)
+    )
+
     let mustAcknowledgeGenericParticipationWarning
-    $: mustAcknowledgeGenericParticipationWarning = isAccountStaked(accountId) && isStakingPossible($stakingEventState)
+    $: mustAcknowledgeGenericParticipationWarning =
+        (isAccountStaked(accountId) && isStakingPossible($stakingEventState)) || isAccountVoting
+
+    let activeParticipationType
+    $: {
+        if (isAccountStaked(accountId) && isAccountVoting) {
+            activeParticipationType = 'stakeVote'
+        } else if (isAccountStaked(accountId)) {
+            activeParticipationType = 'stake'
+        } else if (isAccountVoting) {
+            activeParticipationType = 'vote'
+        } else {
+            activeParticipationType = ''
+        }
+    }
 
     let mustAcknowledgeBelowMinRewardParticipationWarning
     $: {
```

Comment:
This should be defined similarly to `mustAcknowledgeGenericParticipationWarning` (without the curly brackets) and ideally be grouped together with `mustAcknowledgeGenericParticipationWarning`